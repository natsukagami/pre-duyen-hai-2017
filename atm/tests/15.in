730 194
-685340684705675377428415960352728237007424250943422982755880870216589646573050044028410788010314934852683629084710941859094327261643280845219066493346023681258482602543287851878076958051149063668544015407118307881676829859627637071388746030886684320513053353751178793155749378407272884231671187130078307420298724936864800652171230086533526545262070282395083737000626155381982771995536540093058330475254040217942194604543981392614820449525802316719993432749451591678424508178555081179178791780732445617273415307879866752150236066579297142251748648486916479055812138323223460410137142402275917270147239632301691163823633769922040825353247820090012856406192070841289639417798923035087360298217407071405877424955796267247290088539940
