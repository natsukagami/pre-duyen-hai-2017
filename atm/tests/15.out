99999999999998837881676829859627637071388746030886684320513053353751178793155749378407272884231671187130078307420298724936864800652171230086533526545262070282395083737000626155381982771995536540093058330475254040217942194604543981392614820449525802316719993432749451591678424508178555081179178791780732445617273415307879866752150236066579297142251748648486916479055812138323223460410137142402275917270147239632301691163823633769922040825353247820090012856406192070841289639417798923035087360298217407071405877424955796267247290088539940
